common/crypto/attestation-api/test/verify_evidence_app_go/main.go

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
i guess that file is obsolete? (only go program and refers to FPC version of attestation-api which is different?

Reply:
Per conversations in the PDO community meeting, I have removed the dead Go code.